src/Tribe/Tickets.php

```diff
@@ -680,6 +680,19 @@ public function post_has_tickets( $post ) {
 		 * @return Tribe__Tickets__Ticket_Object[] List of ticket objects.
 		 */
 		public function get_tickets( $post_id ) {
+
+			/* @var Tribe__Cache $cache */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: d02cd7634eb74f27d0b0c173785c20f6</sub>